test/python/quantum_info/operators/symplectic/test_sparse_pauli_op.py

```diff
@@ -462,6 +462,24 @@ def test_simplify2(self, num_qubits, num_adds):
         np.testing.assert_array_equal(spp_op.paulis.phase, np.zeros(spp_op.size))
         np.testing.assert_array_equal(simplified_op.paulis.phase, np.zeros(simplified_op.size))
 
+    def test_chop(self):
+        """Test chop, which individually truncates real and imaginary parts of the coeffs."""
+        eps = 1e-10
+        op = SparsePauliOp(
+            ["Z", "Z", "X", "Y"], coeffs=[eps + 1j * eps, 1 + 1j * eps, eps + 1j, 1 + 1j]
+        )
+        simplified = op.chop(tol=eps)
+        expected_coeffs = [1, 1j, 1 + 1j]
+        self.assertListEqual(simplified.coeffs.tolist(), expected_coeffs)
```

Comment:
This should assert that the correct symplectic operators are present as well - it'd be easy for them to get out-of-sync when there's a deletion.

Reply:
added 👍🏻